### PR TITLE
WIP - sudo prompt

### PR DESCRIPTION
**Pull request by mvollmer**
First sparks: https://www.youtube.com/watch?v=Xi_anqItZaU&feature=youtu.be

 * [ ] As noted in #13222 we should add a test for privileged -> drop privileges -> privileges.

### CODE REVIEW

**Comment by mvollmer** [comment]
(I have not yet been able to reproduce the bug in the video... maybe it just doesn't like to be recorded.)

**Comment by marusak** [comment]
Nice :)
I thought that all buttons/actions would be available, but if some action would require root, it would just prompt for it. But it maybe would be problematic if something would be happening asynchronously and would require root privileges?

(and I like your BJ-Martin and BJ-Lars bookmarks :D I always need to go to calendar when some other meeting than daily standup is happening as I never remember those different urls)

**Comment by mvollmer** [comment]
> I thought that all buttons/actions would be available, but if some action would require root, it would just prompt for it. But it maybe would be problematic if something would be happening asynchronously and would require root privileges?

That's all still up for discussion.

File: src/bridge/cockpitrouter.c [hunk 1]
```diff
@@ -1133,6 +1151,7 @@ cockpit_router_add_bridge (CockpitRouter *self,
   if (!output->next)
     {
       rule->callback = process_open_peer;
+      rule->reset_failed = process_reset_peer_if_idle;
```
Comment (mvollmer) [inline src/bridge/cockpitrouter.c]: We need to set reset_failed for other rules as well.

**Comment by mvollmer** [comment]
This is obsoleted by #13482

**Pull request closed without merging**